Resources/Audio/_Goobstation/Ambience/Antag/attributions.yml

```
@@ -7,6 +7,11 @@
   license: "CC-BY-NC-SA-4.0"
   copyright: "Used with permission from the creator, Cuboos"
   source: "https://soundcloud.com/cuboos/engrev?in=cuboos/sets/space-station-13-audio-logs"
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)